services/core-web/common/actions/authenticationActions.js

```diff
@@ -11,13 +11,6 @@ export const logoutUser = () => ({
   type: ActionTypes.LOGOUT,
 });
 
-export const storeKeycloakData = (data = {}) => ({
```

Comment:
other modules can access keycloak data via useKeycloak function, no need for this action.